appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/resource/allocator/ConnectorAllocator.java

```diff
@@ -143,30 +140,14 @@ public ConnectorAllocator(PoolManager poolMgr,
 
 
     @Override
-    public ResourceHandle createResource()
-            throws PoolingException {
+    public ResourceHandle createResource() throws PoolingException {
         try {
-            ManagedConnection mc =
-                    mcf.createManagedConnection(subject, reqInfo);
-
-            ResourceHandle resource =
-                    createResourceHandle(mc, spec, this, info);
-            ConnectionEventListener l =
-                    new ConnectionListenerImpl(resource);
+            ManagedConnection mc = mcf.createManagedConnection(subject, reqInfo);
+            ResourceHandle resource = createResourceHandle(mc, spec, this, info);
+            ConnectionEventListener l = new ConnectionListenerImpl(resource);
             mc.addConnectionEventListener(l);
             return resource;
         } catch (ResourceException ex) {
-            Object[] params = new Object[]{spec.getPoolInfo(), ex.toString()};
-            _logger.log(Level.WARNING,"poolmgr.create_resource_error",params);
-            if(_logger.isLoggable(Level.FINE)) {
-                _logger.log(Level.FINE,"Resource Exception while creating resource",ex);
-            }
-
-            if (ex.getLinkedException() != null) {
```

Comment:
This is obsoleted deprecated getter, generally not supported any more, and also ignored on other places. 
TODO in next commits of this PR: Remove all calls of the setter.